mlflow/spark.py

```diff
@@ -573,6 +611,18 @@ def _validate_model(spark_model):
         )
 
 
+def _is_spark_connect_model(spark_model):
```

Comment:
Can we use `isinstance(spark_model, ConnectModel)`?

Reply:
We can . But for pyspark < 3.5, importing `pyspark.ml.connect` will fail, so checking class module makes code simpler

Reply:
Capturing importing error seems more straightforward to me.

Reply:
Is `"pyspark.ml.connect.foo.bar"` a spark connect model?

Reply:
Yes `isinstance(spark_model, ConnectModel)` is safer.